app/templates/detail.hbs

```diff
@@ -48,64 +48,42 @@
       </c.header>
       <c.content>
 
-        {{#if this.model.session.approvedDate}}
-          <section
-            class="au-u-flex au-u-muted au-u-margin-top-tiny"
-            style=" align-items: center;"
-          >
-            <AuIcon
-              class="au-u-margin-right-small"
-              @size="large"
-              @icon="check"
-            />
-            <p>
-              Goedgekeurd -
-              {{iso-formatter this.model.session.approvedDate}}
-
-            </p>
-          </section>
-        {{/if}}
-
-        {{#if this.model.municipality}}
-          <section
-            class="au-u-flex au-u-muted au-u-margin-top-tiny"
-            style=" align-items: center;"
-          >
-            <AuIcon
-              class="au-u-margin-right-small"
-              @size="large"
-              @icon="check"
-            />
-            <p>{{this.model.title_agenda.value}}
-              -
-              {{#if this.model.geplandeStart.value}}
-                {{iso-formatter this.model.geplandeStart.value}}
-              {{else}}
-                Geen Datum
-              {{/if}}</p>
-          </section>
-          <section
-            class="au-u-flex au-u-muted au-u-margin-top-tiny"
-            style=" align-items: center;"
-          >
+        <section
+          class="au-u-flex au-u-muted au-u-margin-top-tiny"
+          style=" align-items: center;"
+        >
+          <AuIcon class="au-u-margin-right-small" @size="large" @icon="check" />
+          <p>
+            {{#if this.model.session.startedAt}}
+              {{iso-formatter this.model.session.startedAt}}
+            {{else}}
+              Geen Datum
+            {{/if}}</p>
+        </section>
+        <section
+          class="au-u-flex au-u-muted au-u-margin-top-tiny"
+          style=" align-items: center;"
+        >
+          {{#if this.model.session.governingBody.administrativeUnit.name}}
             <AuIcon
               class="au-u-margin-right-small"
               @size="large"
               @icon="message"
             />
             <LinkTo
               @route="municipality"
-              @model={{this.model.eenheid_werkingsgebied_label.value}}
+              @model={{this.model.session.governingBody.administrativeUnit.name}}
```

Comment:
```suggestion
              @model={{this.model.session.municipality}}
```

Reply:
You can directly use the session.municipality here, make it more readable and handle default value when value is missing.